repos.d/managarm.yaml

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+# Managarm
+###########################################################################
+
+- name: managarm_x86_64
+  type: repository
+  desc: Managarm x86_64
+  family: managarm
+  color: "97bde3"
+  minpackages: 240
+  default_maintainer: fallback-mnt-managarm@repology
+  sources:
+    - name: [ x86_64 ]
+      fetcher:
+        class: TarFetcher
+        url: 'https://builds.managarm.org/repos/packages/managarm/rolling/{source}-repodata'
+      parser:
+        class: ManagarmPlistParser
+      subrepo: '{source}'
+  repolinks:
+    - desc: Managarm home
+      url: https://managarm.org
+    - desc: Managarm build server
+      url: https://builds.managarm.org
+  packagelinks:
+    - type: PACKAGE_RECIPE
+      url: 'https://github.com/managarm/bootstrap-managarm'
```

Comment:
This is supposed to refer to a specific package recipe, usually an url template with `{srcname}` substitution.

Reply:
At the moment, a package recipe file can not be generated based on information available in the XBPS repo. I'll see if I can make something work, but at the moment recipe files are a group of .yaml files, named (approximately) by which gentoo package category they would belong to.

Reply:
I don't think that we would be able to do this with the setup we currently have. Best we could do with that is to link to the correct file, which contains a number of recipes (all of that gentoo-style category).

Reply:
Maybe we could also have links to the build log of the package?
It wouldn't be impossible to have XBPS add a link to the package metadata / redirect to the line in the github repo either, although that would all probably only begin working when packages are rebuilt (which, to be fair, happens quite often) and/or require updates to XBPS

Reply:
> I don't think that we would be able to do this with the setup we currently have

Unfortunately, recipe links are required.

> Maybe we could also have links to the build log of the package?

Sure, this is always welcome.